cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -7,7 +7,15 @@ class HttpCache
 
   # Language header and cookie are needed to separately cache language-specific pages.
   LANGUAGE_HEADER = %w(Accept-Language)
-  LANGUAGE_COOKIES = %w(language_ pm)
+
+  DEFAULT_COOKIES = [
+    # Language drop-down selection.
+    'language_',
+    # Page mode, for A/B experiments and feature-flag rollouts.
+    'pm',
```

Comment:
Ah, of course.  We can definitely use this by having a few internal users manually set a `pm` cookie so that they see the new header work before anybody else.  Thanks.